addons/payment_paypal/static/src/js/payment_form.js

```diff
@@ -134,6 +135,7 @@ paymentForm.include({
         await rpc('/payment/paypal/complete_order', {
             'provider_id': provider_id,
             'order_id': orderID,
+            'tx_ref': this.paypalData[this.selectedOptionId].paypalTxRef,
```

Comment:
The controller expects `reference`
```suggestion
            'reference': this.paypalData[this.selectedOptionId].paypalTxRef,
```